packages/react/src/runtime/css-custom-property.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns a CSS custom property value with an optional suffix & prefix.
+ * Prefix will only be added if there is a suffix.
+ * If the value is undefined a fallback value will be returned to prevent children inheriting parent values.
+ *
+ * @param value
+ * @param suffix
+ * @param prefix
+ */
+export default function cssCustomPropertyValue(
```

Comment:
Do we need to do some perf testing here since we introduce this during runtime?

Reply:
Nevermind. It won't be that slow..hehe. Few if checks only.

Reply:
😄